client/assets/JavaScript/thought.js

```diff
@@ -0,0 +1,44 @@
+function createUrlQuery() {
+  const title = window.location.href.split("???")[1];
+  const url = `http://localhost:3000/post/title/${title}`;
+  console.log(url);
+  return url;
+}
+
+async function fetchData() {
+  const url = createUrlQuery();
+  const fetchResults = await fetch(url)
+    .then((res) => res.json())
```

Comment:
2. I think this combination of await and then may be failing to deliver the json response.